calico_containers/calico_ctl/connectors.py

```diff
@@ -32,6 +35,19 @@
                                                 etcd_authority))
     sys.exit(1)
 
+# Check etcd environment values for etcd with SSL/TLS
+etcd_scheme = os.getenv(ETCD_SCHEME_ENV, ETCD_SCHEME_DEFAULT)
+etcd_key_file = os.getenv(ETCD_KEY_FILE_ENV, "")
+etcd_cert_file = os.getenv(ETCD_CERT_FILE_ENV, "")
+etcd_ca_cert_file = os.getenv(ETCD_CA_CERT_FILE_ENV, "")
+
+try:
+    assert_valid_etcd_keys(etcd_scheme, etcd_key_file, etcd_cert_file,
```

Comment:
Do we need to do this here?  Can't we just catch the exception from connecting to the datastore?